### ISSUE TITLE
is calling a class in the side menu possible?

### ISSUE THREAD

**Issue by pixielot** [adamrocker/simple-side-drawer#4]
Hello, I'm trying to call a different class within the side menu layout. Is that possible?

**Comment by adamrocker** [comment]
Yes, it is. You should refer the view which is located in side-menu layout by using findViewById() method. You can see it in sample application :)

**Comment by shashidhar502** [comment]
Can i slide to particluar position and then comeback to previous position the 2 xmls has to be slide
